### PR TITLE
planning: modified lane_change process() function

### PR DESCRIPTION
**Pull request by sjiang2018**

### CODE REVIEW

**Comment by panjiacheng** [comment]
Why remove all the ADEBUG messages?

File: modules/planning/tasks/deciders/lane_change_decider/lane_change_decider.cc [hunk 1]
```diff
@@ -200,13 +208,16 @@ void LaneChangeDecider::PrioritizeChangeLane(
...
+  return;
+  /* disable rest to avoid core dump */
+  ADEBUG << "removed change lane";
   auto iter = reference_line_info->begin();
   while (iter != reference_line_info->end()) {
     if (iter->IsChangeLanePath()) {
-      iter = reference_line_info->erase(iter);
-    } else {
-      ++iter;
+      reference_line_info->erase(iter);
```
Comment (JasonXu12) [inline modules/planning/tasks/deciders/lane_change_decider/lane_change_decider.cc]: Please don't change here as there might be more than one lane_change reference_line in the list to remove.

**Comment by JasonXu12** [comment]
> Why remove all the ADEBUG messages?
> I guess you can leave them there if they are helpful.

**Comment by JasonXu12** [comment]
> > Why remove all the ADEBUG messages?
> > I guess you can leave them there if they are helpful.
Most of the messages can be found from planning context ChangeLaneStatus message. No need to do the duplicate logging here.

**Pull request closed without merging**